test/test.js

```diff
@@ -51,7 +51,7 @@ test('markdown', (t) => {
     t.equal(result.source, fixture, 'filename');
     t.deepEqual(_.orderBy(result.warnings, ['line', 'column']), markdownExpectedWarnings);
     t.end();
-  }).catch(t.end);
+  }).catch(t.threw);
```

Comment:
Oh, I guess maybe this is different for tape vs. tap, and I should be doing this the tap way. Thanks.